README.md

```diff
@@ -71,6 +71,8 @@ most common customization and extension scenarios:
   library](./docs/trace/extending-the-sdk/README.md#instrumentation-library)
```

Comment:
Not directly related to this PR, we probably need to move instrumentation library to its own docs - the current one is very tight to distributed tracing.

Reply:
Yes agree.